.stylelintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": ["@edx/stylelint-config-edx"],
+  "rules": {
+    "selector-pseudo-class-no-unknown": [true, {
+      "ignorePseudoClasses": ["export"]
+    }],
+    "unit-no-unknown": [true, {
+      "ignoreUnits": ["\\.5"]
+    }],
+    "property-no-vendor-prefix": [true, {
+      "ignoreProperties": ["animation", "filter", "transform", "transition"]
+    }],
+    "value-no-vendor-prefix": [true, {
+      "ignoreValues": ["fill-available"]
+    }],
+    "function-no-unknown": null,
+    "number-leading-zero": "never",
+    "no-descending-specificity": null,
+    "selector-class-pattern": null,
+    "scss/no-global-function-names": null,
+    "color-hex-case": "upper",
+    "color-hex-length": "long",
+    "scss/dollar-variable-empty-line-before": null,
+    "scss/dollar-variable-colon-space-after": "at-least-one-space",
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "scss/at-import-partial-extension": null,
+    "scss/comment-no-empty": null,
+    "import-notation": "string",
+    "property-no-unknown": [true, {
+      "ignoreProperties": ["xs", "sm", "md", "lg", "xl", "xxl"]
+    }],
+    "alpha-value-notation": "number"
+  }
+}
```

Comment:
Similar to my comment on `.eslintrc.js` (and also non-blocking) - should these be MFE specific overrides instead of PRs to https://github.com/openedx/stylelint-config-edx?

Reply:
This configuration was also taken from [Authoring MFE](https://github.com/openedx/frontend-app-authoring/blob/master/.stylelintrc.json). I completely agree with your opinion that our MFEs should consume most of the necessary linter configuration rules from shared configs. But it seems to me that this should be done in a more global format by checking all MFEs and unifying the configurations.